content/en/agent/logs/_index.md

```diff
@@ -90,6 +90,10 @@ logs:
 
 If you are using Serilog, `Serilog.Sinks.Network` is an option for connecting with UDP.
 
+As of Agent version 7.31.0, the agent will keep a TCP connection open indefinitely.
+Previous versions closed conections after 60 seconds without log messages.
```

Comment:
This was not documented before; let's not document it now.

```suggestion
```

Reply:
The sentence before is pretty vague without this counterpoint.  Maybe I could add " open indefinitely, even if idle."?

Or, maybe this doesn't need to be documented at all, if what we think of as "removing a feature" is actually "removing a bug" from the user's perspective. WDYT?

Reply:
I think if we rephrased the first sentence as "In the Agent version 7.31.0+, the TCP connection stays open indefinitely even when idle." we can delete the rest of this!